skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h

```diff
@@ -3,15 +3,46 @@
 
 struct BaseActorValues
 {
+   
   float health = 100;
   float stamina = 100;
   float magicka = 100;
   float healRate = 100;
```

Comment:
100 is too much for rates. see https://elderscrolls.fandom.com/wiki/Console_Commands_(Skyrim)/ActorValues

Reply:
Done @Pospelove